test/models/parser/tiktok_profile_test.rb

```diff
@@ -44,17 +44,23 @@ def doc
 
     assert_equal '@fakeaccount', data['external_id']
     assert_equal '@fakeaccount', data['username']
-    assert_match /Huxley the Panda Puppy🐼🐶\(pandaloon \(@huxleythepandapuppy\) on TikTok/, data['description']
+    assert_match '@fakeaccount', data['title']
```

Comment:
I think we should remove these lines that were added in this test. In general we try to keep our integration tests fairly minimal because there's no guarantee that they'll fetch the HTML and parse successfully. As a result, we try to only test what will be consistent across failure and success -- in this case, it's the things we always pull from the URL (not title, description, or author name, which we try to pull from the HTML)

Instead, we try to test all of the values (fallback and also successful parse) in the unit tests below.

Reply:
Also, I can't request changes because it's my PR. Learning about Github permissions 😅